packages/scandipwa/src/component/MyAccountAddressPopup/MyAccountAddressPopup.container.js

```diff
@@ -116,7 +116,7 @@ export class MyAccountAddressPopupContainer extends PureComponent {
 
     showAddressNotification(status, operation) {
         const { showSuccessNotification, showErrorNotification } = this.props;
-        const message = __('You %s the address', operation);
+        const message = __('You saved the address', operation);
```

Comment:
1. `%s` is a placeholder to put the variable in the translation string. In this case, it's an operation variable.
2. Depending on the condition, it may have different values resulting in different notification messages.
3. You have hardcoded the string, so for certain conditions, it will produce the wrong notification.
4. In fact, you should investigate why the `operation` variable is `undefined` and pass the correct value.